src/About.js

```diff
@@ -205,23 +197,17 @@ class AboutComponent extends React.Component {
                 t={t}
                 i18nKey="about.confessit_is_open_source"
                 components={{
-                  i: <i className="fa fa-github" />,
+                  githubicon: <i className="fa fa-github" />,
```

Comment:
You can't rename components here without also adjusting the name used in translation files. 

https://github.com/kas-catholic/confessit-web/blob/59c0272fb3f2e2e660e141b8fd210b224c562793/public/locales/en/translation.json#L60

Note that this string correctly shows the icon on the about page at https://confessit.app but is broken on the deploy preview at https://deploy-preview-64--confessit-web.netlify.app/

Reply:
you can even see, in the list of **commits** for this PR, that there is a **referenced commit**: the commit that takes care of this problem in PR #60 references the current PR, so that you can see even within the current PR that it will be taken care of in PR #60...

Reply:
I also mentioned this exact situation in a comment in the current PR... https://github.com/kas-catholic/confessit-web/pull/64#issuecomment-1560216632